cloud/testserver/cloudstore/store.go

```diff
@@ -152,17 +152,21 @@ const (
 )
 
 // LoadDatastore loads the data store from a JSON file.
-func LoadDatastore(fpath string) (*Data, error) {
+func LoadDatastore(fpath string) (store *Data, defaultOrg string, err error) {
 	data, err := os.ReadFile(fpath)
 	if err != nil {
-		return nil, errors.E(err, "failed to read testserver data")
+		return nil, "", errors.E(err, "failed to read testserver data")
 	}
-	var dstore Data
+	type serverData struct {
+		DefaultOrg string `json:"default_org"`
```

Comment:
Instead of being the default org, I'd name it selected one:
```suggestion
		SelectedOrg string `json:"selected_org"`
```
What do you think? It's more accurate in my opinion with the use case.

Reply:
the idea is that this is the "default org used in tests". I shorted it to "default_org".
Using "selected_org" would make more sense if this field were picked up by the CLI itself, I think.

Reply:
I changed to `default_test_org` to make the intention more clear. As I said, if using `selected_org` imply this is picked by CLI automatically or only this one is used which is not true, some tests use a different org.